packages/trader/src/Modules/Contract/Containers/contract-replay.jsx

```diff
@@ -10,7 +10,10 @@ import Icon              from 'Assets/icon.jsx';
 import AppRoutes         from 'Constants/routes';
 import { connect }       from 'Stores/connect';
 
-const SmartChart = React.lazy(() => import(/* webpackChunkName: "smart_chart" */'../../SmartChart'));
+const SmartChart               = React.lazy(() => import(/* webpackChunkName: "smart_chart", webpackPreload: true */'../../SmartChart'));
```

Comment:
can we load smartchart from the `<Lazy />` component too? 🤔

Reply:
In theory we can, but I wasn't sure of the side-effects, especially in trading page. So I left it as it is.

Reply:
In our multiple-contracts PR, we're going to be testing to see if this improves performance or not. So we can handle it there instead.